src/controllers/request.ts

```diff
@@ -57,13 +50,23 @@ export const createRequest = async (
   information: string
 ) => {
   try {
+    const seller = await Service.findOne({ _id: service });
+
+    if (!seller) {
+      return {
+        status: false,
+      };
+    }
+
     const request = new Request({
       service,
       buyer,
+      seller: seller?._id,
```

Comment:
and here you're saying seller._id but seller is of `Service.findOne()`

also since you guaranteed existence of seller with the !seller check, you can change it from seller?._id to seller!._id

Reply:
Fixed in Commit